grails-app/conf/spring/resources.groovy

```diff
@@ -11,6 +12,9 @@ beans = {
     hpoAssociationFactory(HpoAssociationFactory, hpoOntology)
     hpoAssociation(hpoAssociationFactory: "getInstance")
 
+    hpoLoincFactory(Loinc2HpoFactory)
```

Comment:
Create a loinc factory bean. Then create the hpoLoinc injectable bean with our data by calling the previous beans getinstance method which returns a Loinc2Hpo Instance... all only once on startup of the app.

Reply:
This is nice!